src/chrome/content/rules/Engadget.com.xml

```diff
@@ -44,10 +30,6 @@
 	<securecookie host="^\." name="^gdgt_edition$" />
 	<securecookie host="^\w" name=".+" />
```

Comment:
Please remove the obsolete comment and replace the cookie rules with a trivial one `<securecookie host=".+" name=".+" />`